src/components/FeedbackError/FeedbackError.tsx

```diff
@@ -1,29 +1,34 @@
+import { ReactNode } from 'react';
+
 import { AnimatePresence, motion } from 'framer-motion';
-import { useError } from 'stores/useError/useError';
+
+import { useError } from '~stores/useError/useError';
 
 import Icon from '~components/Icon/Icon';
 
 import scss from './FeedbackError.module.scss';
 
 import { animationVariants } from './FeedbackError.data';
 
-function FeedbackError() {
+function FeedbackError(): ReactNode {
   const errorMessage = useError((state) => state.errorMessage);
 
-  const renderError = () => (
+  const renderError = (): ReactNode => (
     <motion.div
-      initial="hidden"
       animate="visible"
-      exit="hidden"
       className={scss.wrapper}
-      variants={animationVariants}
       data-testid="error-container"
+      exit="hidden"
+      initial="hidden"
+      variants={animationVariants}
     >
-      <Icon icon="error" size="large" className={scss.errorIcon} />
+      <Icon className={scss.errorIcon} icon="error" size="large" />
       <p className={scss.errorMessage}>{errorMessage}</p>
     </motion.div>
   );
-  return <AnimatePresence>{errorMessage && renderError()}</AnimatePresence>;
+  return (
+    <AnimatePresence>{errorMessage ? renderError() : null}</AnimatePresence>
+  );
```

Comment:
o que te levou a fazer essa mudança? no fim os dois não fazem a mesma coisa?

Reply:
Man se eu te falar q isso foi o eslint fix ce acredita ? kjkj

Reply:
Inclusive fazer ? : null é melhor do que &&

https://medium.com/geekculture/stop-using-for-conditional-rendering-in-react-a0f7b96200f8